core/tests/karma-globals.js

```diff
@@ -21,3 +21,23 @@ var GLOBALS = {
   GADGET_SPECS: {},
   PANEL_SPECS: {}
 };
+
+// jscs:disable maximumLineLength
+/* This function overwrites the translationProvider for a dummy function
+ * (customLoader). This is necessary to prevent the js test warnings about an
+ * "unexpected GET request" when the translationProvider tries to load the
+ * translation files.
+ * More info in the angular-translate documentation
+ * http://angular-translate.github.io/docs/#/guide/22_unit-testing-with-angular-translate
+ */
+// jscs:enable maximumLineLength
+GLOBALS.OVERWRITE_TRANSLATOR_PROVIDER = function($provide, $translateProvider) {
```

Comment:
I'd name this something like TRANSLATOR_PROVIDER_FOR_TESTS, I think that would be clearer. (I was confused by the semantics when I first saw OVERWRITE_TRANSLATOR_PROVIDER.)